force-app/main/default/classes/PrimaryContactController.cls

```diff
@@ -0,0 +1,73 @@
+/*************************************************************************************
+ * @Name         : PrimaryContactController.cls
+ * @Description  : Apex class for PrimaryContact page
+ * @Created By   : Cem
+ * @Created Date : 07.09.2021
+ *************************************************************************************/
+public class PrimaryContactController {
+    
+    public List<contact> contacts {get;set;}
+    
+    public String searchText {
+        get {
+            if(searchText == null) {
+                searchText = ''; 
+            }
+            return searchText;
+        }
+        set;
+    }
+    public Id primaryContactId {get;set;}
+
+    @TestVisible
+    private Account accountRecord;
+
+    //Constructing extension controller for standard account controller
+    public PrimaryContactController(ApexPages.StandardController stdCont) {
+        this.accountRecord = (Account)stdCont.getRecord();
+        this.getRelatedContacts();
+        
+    }
+    
+    
+    public void getRelatedContacts() {
+        this.contacts = new List<Contact>();
+        String formattedSearch = '%'+this.searchText+'%';
+        try {
+            this.contacts  = [SELECT id, LastName, Title, Phone, Is_Primary_Contact__c 
+                FROM Contact 
+                WHERE accountid = :this.accountRecord.id 
+                AND Name LIKE :formattedSearch
+                ORDER BY LastName 
+                LIMIT 990];
+        } catch(Exception ex) {
+            System.debug(ex.getMessage());
+            //use apex page messages to show something like "An error happened, please check your logs"
+            ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.ERROR,'Test'));
```

Comment:
how about this comment and error message?
Also, please put all "texts" into Constant (public static final String CAPITAL_LETTERS_NAME = 'string';) or as a custom label